Source/JavaScriptCore/tools/JSDollarVM.cpp

```diff
@@ -744,15 +744,11 @@ JSC_DEFINE_CUSTOM_SETTER(testStaticAccessorPutter, (JSGlobalObject* globalObject
     return thisObject->putDirect(vm, PropertyName(Identifier::fromString(vm, "testField"_s)), JSValue::decode(value));
 }
 
-static const struct CompactHashIndex staticCustomAccessorTableIndex[9] = {
-    { -1, -1 },
+static const struct CompactHashIndex staticCustomAccessorTableIndex[5] = {
```

Comment:
These table should be different in macOS and others, right? Even if the content is the same (due to length threshold), we should separate via `PLATFORM(MAC)` and have two since it is confusing.